lms/static/js/i18n/fr/djangojs.js

```diff
@@ -659,6 +659,7 @@
     "Endorse": "Approuver", 
     "Ends {end}": "Finit \u00e0 {end}", 
     "Engage with posts": "Participez \u00e0 la conversation", 
+    "Enrol Now": "Inscrivez-vous maintenant", 
```

Comment:
the file cms/static/js/i18n/fr/djangojs.js is missing

Reply:
Done